### DISCUSSION TITLE
Metrics for Individual classes

### DISCUSSION THREAD

**Discussion by Raknash3** [ultralytics/yolov5#3891] [Q&A]
Is it possible to view metrics(Precision, recall and mAP) for individual classes  ?

**Comment by glenn-jocher** [comment]
@Raknash3 these should display automatically. If you not you can pass the --verbose flag:
```
python test.py --verbose
```

**Reply by Raknash3** [reply]
I trained your model on a custom dataset and metrics for individual classes didn't display automatically. Is it possible to use the --verbose flag with train.py  ?

**Reply by Raknash3** [reply]
Ok it works, thank you.
